<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickStitchWindow</class>
 <widget class="QMainWindow" name="QuickStitchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>643</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>816</width>
    <height>643</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Quick Stitch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="imageListWidget">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::CopyAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="hStitchButton">
        <property name="text">
         <string>Horizontal stitch </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="vStitchButton">
        <property name="text">
         <string>Vertical stitch </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>optionMenu</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddImages"/>
   <addaction name="actionRemoveImages"/>
   <addaction name="actionMoveUp"/>
   <addaction name="actionMoveDown"/>
   <addaction name="actionBackground_color"/>
  </widget>
  <action name="actionAddImages">
   <property name="text">
    <string>Add images</string>
   </property>
   <property name="toolTip">
    <string>add multiple images to list</string>
   </property>
  </action>
  <action name="actionRemoveImages">
   <property name="text">
    <string>Remove images</string>
   </property>
   <property name="toolTip">
    <string>remove images</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="text">
    <string>Move up</string>
   </property>
   <property name="toolTip">
    <string>move selected image to up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="text">
    <string>Move down</string>
   </property>
   <property name="toolTip">
    <string>move selecetd image to down</string>
   </property>
  </action>
  <action name="actionBackground_color">
   <property name="text">
    <string>Background color</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
